fe/src/components/user/donators.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <table class="table-auto mt-6">
+    <thead class="border-b border-primary-10">
+      <tr class="flex h-10 items-center">
+        <th class="text-left px-4 w-72">Donator</th>
+        <th class="text-left px-4 flex-1">Campaign Title</th>
+        <th class="text-left px-4 w-40">Amount</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr
+        v-for="{ id, avatar, donator, title, amount } in donors"
+        :key="id"
+        class="flex items-center min-h-[56px] border-b border-disabled"
+      >
+        <td class="px-4 w-72">
+          <div class="flex items-center space-x-2">
+            <span>{{ id }}.</span>
```

Comment:
Maybe you can use `index + 1` instead of `id`? Because `id` may skip numbers and not start at 1
https://vuejs.org/guide/essentials/list.html#v-for